frontend/packages/kubevirt-plugin/integration-tests/tests/vmtemplate.resources.scenario.ts

```diff
@@ -33,7 +36,9 @@ describe('Test adding/removing discs/nics to/from a VM template', () => {
     beforeAll(async () => {
       await vmt.addDisk(hddDisk);
       await vmt.addNIC(multusNetworkInterface);
-      await vm.create();
+
+      await vmt.createVMFromRowLink();
```

Comment:
do we need the whole template every time we want to create a VM from it. Can we utilize the create method to open the wizard (either from template or plain wizard) and process all the steps?

The create processes additional attributes like startOnCreate